### PR TITLE
Update traffic mirroring build to use kernel version v5.15

### PR DESCRIPTION
**Pull request by sanfern**
This PR to build traffic mirroring with kernel v5.15

### CODE REVIEW

File: traffic-mirroring/Makefile [hunk 1]
```diff
@@ -7,106 +6,224 @@ TOOLS_PATH := $(BPF_SAMPLES_PATH)/../../tools
...
+#ifeq ($(VMLINUX_H),)
+#	$(Q)$(BPFTOOL) btf dump file $(VMLINUX_BTF) format c > $@
+#else
+#	$(Q)cp "$(VMLINUX_H)" $@
+#endif
+
+#ifeq ($(VMLINUX_BTF),)
+#	$(error Cannot find a vmlinux for VMLINUX_BTF at any of "$(VMLINUX_BTF_PATHS)",\
+#		build the kernel or set VMLINUX_BTF variable)
+#endif
```
Comment (dthaler) [inline traffic-mirroring/Makefile:180]: Remove commented out code

File: traffic-mirroring/Makefile [hunk 2]
```diff
@@ -1,4 +1,3 @@
-# Copyright Contributors to the L3AF Project.
```
[the remark this answers is not in the document]
Comment (sanfern) [reply traffic-mirroring/Makefile:1]: Makefile originated from kernel source repo. I feel it's not a good idea to claim copyrights on this file.

**Review by jaysheth2 — APPROVED** [review]

**Review by charleskbliu0 — APPROVED** [review]
Looks good

**Comment by sanfern** [comment]
This PR depends on https://github.com/l3af-project/l3afd/pull/245

**Pull request merged**